applications/nrf5340_audio/src/bluetooth/bt_management/scanning/Kconfig

```diff
@@ -25,11 +25,16 @@ menu "Connection"
 
 config BLE_ACL_CONN_INTERVAL
 	int "Bluetooth LE ACL Connection Interval (x*1.25ms)"
-	default 32
+	default 64
 	help
-	  When the LE Audio Controller Subsystem for nRF53 is used, this
-	  interval should be a multiple of the ISO interval and maximum 4x
-	  larger than the lowest interval.
+	  This interval should be a multiple of the ISO interval.
+
+config BLE_ACL_CONN_INTERVAL_FAST
```

Comment:
Do we have to make BLE_ACL_CONN_INTERVAL_FAST a Kconfig entry? Can we instead use SDU interval?

Reply:
Yes, can do that.